src/commands/up.ts

```diff
@@ -100,19 +104,29 @@ export default class Up extends Command {
                 this.log(
                     "Running services in development mode - watching for changes.\n"
                 );
-                this.log(
-                    "Non-Node Applications: Sync changes, Run: '$ chs-dev reload <serviceName>'\n"
-                );
+                const { services, isJavaPresent, isNodePresent } = this.servicesBuildContext;
+
+                if (isNodePresent) {
+                    this.log(
+                        "Node Applications: Automatically sync changes.\n"
+                    );
+                }
+                if (isJavaPresent) {
```

Comment:
How about those without labels that are non-node and non-java i.e. don't have a builder label or it is repo?

Reply:
Is the plan to address these after Java?

Reply:
This is resolved. They will be handled just as Java. But the reload functionality will be disabled for no builders.